### ISSUE TITLE
surgical coordinates

### ISSUE THREAD

**Issue by saskiad** [AllenNeuralDynamics/aind-data-schema#171]
In procedures, add the lambda-bregma distance, and add something that makes it clear that coordinates are relative to Bregma.

**Comment by saskiad** [comment]
I thought this was going to be easy, but ... in the SWR the coordinates are "relative to Bregma unless noted" and several of the examples I see have "rostral to lambda" in the notes.
So perhaps we need a "coordinate reference" with options of Bregma or Lambda? What do you think @mekhlakapoor @dyf 
Ugh, can't we just use a single reference point people?

**Comment by mekhlakapoor** [comment]
I agree that we should prob have a coordinate reference in the schema, and in my api i can have it default to bregma and write a parsing method to check notes for "lambda"
